firestore-debug.log

```diff
@@ -0,0 +1,23 @@
+Mar 02, 2024 8:26:49 PM com.google.cloud.datastore.emulator.firestore.websocket.WebSocketServer start
```

Comment:
regardless of if we keep firestore emulator, this should be in gitignore right?

Reply:
yep